share/templates/indexer-deployment.yaml

```diff
@@ -62,7 +62,7 @@ spec:
               if [ -f /code/newrelic.ini ]; then
                 PREFIX='newrelic-admin run-program'
               fi
-              $PREFIX gosu www-data sharectl search daemon
+              $PREFIX gosu www-data python manage.py shtrove_indexer_run
```

Comment:
Is this a new management command in the application? If so, please also increment the chart version (in `share/Chart.yaml`) by one Micro version. If this command has existed since at least the ES8 upgrade, then it's backwards-compatible and not required to be incremented.